### PR TITLE
Update docs manifest

### PR DESCRIPTION
**Pull request by willtebbutt**
I don't really understand why the docs failed to build when I merged the last PR into main. They seem to build fine locally. Going to see if just upgrading the manifest sorts stuff out...

### CODE REVIEW

**Comment by willtebbutt** [comment]
Am also trying just re-running the docs build -- they built successfully on the PR, so I don't understand why they failed on main 🤷

File: docs/Manifest.toml [hunk 1]
```diff
@@ -1,31 +1,33 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.2"
+julia_version = "1.8.0-rc3"
```
Comment (tpgillam) [inline docs/Manifest.toml:3]: This is possibly not a good idea.

Comment (tpgillam) [reply docs/Manifest.toml:3]: (In case the manifest format has changed between 1.7 & 1.8. Although I don't think it has)

Comment (willtebbutt) [reply docs/Manifest.toml:3]: Agreed. I'm not entirely sure that we need this PR anyway though

**Comment by willtebbutt** [comment]
Closing because the dev docs appear to have been built successfully -- I misunderstood which bit of CI had failed

**Pull request closed without merging**